lib/Weasel/Widgets/Dojo/Option.pm

```diff
@@ -5,8 +5,8 @@ use strict;
 use warnings;
 
 use Moose;
-use Weasel::Widget::HTML::Selectable;
-extends 'Weasel::Widget::HTML::Selectable';
+use Weasel::Widgets::HTML::Selectable;
```

Comment:
There are 2 commits here: 1 to correct the name of Weasel::Widget to Weasel::Widgets. The other to correct the interaction with the widget. Could you test and PR them separately, please? (Also: did you try if *just* correcting the package names fixes any of the expected behaviour? And if so, maybe document which part of the behaviour then improves?)